README.md

```diff
@@ -1,2 +1,62 @@
+[![Gem Version](https://badge.fury.io/rb/mais-orcid-client.svg)](https://badge.fury.io/rb/mais-orcid-client)
+[![CircleCI](https://circleci.com/gh/sul-dlss/mais-orcid-client.svg?style=svg)](https://circleci.com/gh/sul-dlss/mais-orcid-client)
+[![Maintainability](https://api.codeclimate.com/v1/badges/5919e7ae4cd162861585/maintainability)](https://codeclimate.com/github/sul-dlss/mais-orcid-client/maintainability)
+[![Test Coverage](https://api.codeclimate.com/v1/badges/5919e7ae4cd162861585/test_coverage)](https://codeclimate.com/github/sul-dlss/mais-orcid-client/test_coverage)
+
 # mais-orcid-client
 API client for accessing MAIS's ORCID endpoints.
+
+## Installation
+
+Install the gem and add to the application's Gemfile by executing:
+
+    $ bundle add mais-orcid-client
+
+If bundler is not being used to manage dependencies, install the gem by executing:
+
+    $ gem install mais-orcid-client
+
+## Usage
+
+For one-off requests:
+
+```ruby
+require "mais_orcid/client"
+
+# NOTE: The settings below live in the consumer, not in the gem.
+client = MaisOrcid::Client.configure(
+  client_id: Settings.mais_orcid.client_id,
+  client_secret: Settings.mais_orcid.client_secret,
+  base_url: Settings.mais_orcid.base_url,
+  transfer_endpoint_id: Settings.globus.transfer_endpoint_id
+)
+client.fetch_orcid_user(sunetid: 'nataliex')
+```
```

Comment:
Do we want this gem only for MaIS ORCID functionality or all MaIS functionality, e.g., the workgroups API? cc: @peetucket

Reply:
Since we didn't know (1) whether other MAIS endpoints would require separate credentials and (2) whether we would ever use those other endpoints, I erred on this side of making this client name more descriptive but limited, rather than cursing us to an eternity of "What the heck is the name of that client that does the Stanford / ORCID thing?"

Reply:
Good questions. For (2), @peetucket, how likely do you think we are to need to integrate with the workgroups API in the next year? If likely, it'd be good to look into API details and determine if these two APIs share creds.

Reply:
We decided that it's fine to have `orcid` in the name of the gem. Resolving.